vagrant/provisioning/roles/solr/tasks/configset.yml

```diff
@@ -51,6 +51,15 @@
     dest: "{{ root_folder }}/app/solr/server/solr/configsets/arkcase/conf/"
     mode: 0644
   when: "schema_xml.stat.exists == false or 'AFDP' not in schema_contents.stdout"
+
```

Comment:
On the previous task ("copy schema.xml if needed") you can add another clause to the `when`: `... and not (enable_solr_snippets | default false)`.

Reply:
Updated

Reply:
I added extra parentheses to make the order of operations explicit; and also fixed the octal permission flags due to a warning from ansible-lint.